examples/vision/captcha_ocr.py

```diff
@@ -249,7 +249,7 @@ def build_model():
     x = layers.Bidirectional(layers.LSTM(64, return_sequences=True, dropout=0.25))(x)
 
     # Output layer
-    x = layers.Dense(len(characters) + 1, activation="softmax", name="dense2")(x)
+    x = layers.Dense(len(char_to_num.get_vocabulay()) + 1, activation="softmax", name="dense2")(x)
```

Comment:
Is the +1 still valid then?

Reply:
If I'm not mistaken that Dense layer should always have `len(vocabulary)+1` units (afaik it's because of the pseudo blank character of the ctc layer, but I might be wrong). Previously `char_to_num` vocabulary length was equal to `len(characters)`. 

In addition, if I don't add that `+1`, I get [this error](https://github.com/keras-team/keras-io/issues/514#issuecomment-868954881).

But, I'm new to this ctc stuff, so I might be wrong :/

Reply:
Ok, sounds good. Please replicate the fix in the other 2 files

Reply:
Done!